CHANGELOG.md

```diff
@@ -4,6 +4,41 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.0.1] - 2024-05-14
+
+Props: @GaryJones, @jrnfl, @terriann, @rebeccahum
+
+Increases requirements for PHPCS from 3.7.2 to 3.9.2 for improved PHP 8.2 support. Please ensure you run `composer update automattic/vipwpcs --with-dependencies` to benefit from this.
+
+### Added
+- [#805](https://github.com/Automattic/VIP-Coding-Standards/pull/805): Use PHPCSStandards fork due to squizlabs repo being abandoned.
+
+### Removed
+- [#812](https://github.com/Automattic/VIP-Coding-Standards/pull/812): Functions/RestrictedFunctions: Remove restricting `term_exists()`.
+- [#814](https://github.com/Automattic/VIP-Coding-Standards/pull/814): Functions/RestrictedFunctions: Remove restricting `get_page_by_title()`.
+- [#817](https://github.com/Automattic/VIP-Coding-Standards/pull/817): Functions/RestrictedFunctions: Remove restricting `get_page_by_path()`.
+
+### Changed
+- [#799](https://github.com/Automattic/VIP-Coding-Standards/pull/799): Classes/DeclarationCompatibility: Sync signature definitions with WP Core.
+
+### Maintenance
+- [#797](https://github.com/Automattic/VIP-Coding-Standards/pull/797): GH: Add initial dependabot config file.
+- [#798](https://github.com/Automattic/VIP-Coding-Standards/pull/798): GH Actions: Bump actions/checkout to 4 from 3.
+- [#802](https://github.com/Automattic/VIP-Coding-Standards/pull/802): GH Actions: Minor tweaks of updating an incorrect URL and adding names to steps.
+- [#803](https://github.com/Automattic/VIP-Coding-Standards/pull/802): GH Actions: Builds against PHP 8.3 can no longer fail, but 8.4 can.
+- [#805](https://github.com/Automattic/VIP-Coding-Standards/pull/805): Composer: Raise minimum version for better PHP 8.2 support and update various constraints:
+- Composer:
```

Comment:
If you're going to use grouping here, why not use it for all items in "Maintenance", like was done for the 2.3.4 release ?